lib/puppet-lint/bin.rb

```diff
@@ -69,11 +74,6 @@ def run
         end
       end
       return return_val
-
-    rescue PuppetLint::NoCodeError
-      puts "puppet-lint: no file specified or specified file does not exist"
```

Comment:
If that's the case, should the `begin` and `end` and a level of indentation from L43-77 be removed as well, for clarity?